src/occupational_classification/lookup/soc_lookup.py

```diff
@@ -3,10 +3,16 @@
 rephrased descriptions. It also handles preprocessing of SOC data and provides metadata
 for the classifications.
 
+The data is provided form ONS, located in src/occupational_classification/data.
```

Comment:
should be `from ONS`. Refer to the `docs/index` [file](https://github.com/ONSdigital/soc-classification-library/blob/main/docs/index.md) where you provide URLs to the files.